server/datastore/mysql/migrations/tables/20230413193404_AddSoftwareInstalledPathsTable.go

```diff
@@ -0,0 +1,30 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230413193404, Down_20230413193404)
+}
+
+func Up_20230413193404(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+		CREATE TABLE host_software_installed_paths
+		(
+			host_id        INT UNSIGNED NOT NULL,
+			software_id    INT UNSIGNED NOT NULL,
```

Comment:
I'm guessing that in that scenario we will also have a single row on the on the `host_software` table? Since we don't use the installed path when differentiating software records https://github.com/fleetdm/fleet/pull/11214/files/6640dc0c6d84d9a5a0e1f76c5807493584c4a1b5#diff-4fe70646343c5f7a1b61dd6906889113f0e0ae63f684ab74a617acecb39fe6d3L37